### PR TITLE
Github: templates and new user interface

### PR DESCRIPTION
**Pull request by alwaysintreble**
Add pull request and issue templates to help guide new contributions. Moved contributing and code of conduct out of readme for same reasons. Currently have my fork set up with these changes if you would like to see them in action.

### CODE REVIEW

**Comment by black-sliver** [comment]
I want this, but I think @Ijwu would be the best to review this if possible

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+description: File a bug report.
+title: "Bug: "
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! If this bug occurred during generation check your 
+        Archipelago install for a log and upload it - if relevant - with this report, as well as all yaml files used.
```
Comment (Ijwu) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:11]: Where are the logs? Include that info here.

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+labels:
+  - enhancement
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please replace `Category` in the title with what this feature will be targeting, such as Core generation,
+        website, documentation, or a game.
+        Note: this is not for requesting new games to be added. If you would like to request a game the best place to
+        ask is about it is in the discord.
```
Comment (Ijwu) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:13]: Where in the discord and where is the discord? Provide that info here, please.

File: docs/code_of_conduct.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+We conduct ourselves openly and inclusively here. Please do not contribute to an environment which makes other people uncomfortable. This means that we expect all contributors or participants here to:
+
+* Be welcoming and inclusive in tone and language.
+* Be respectful of others and their abilities.
+* Show empathy when speaking with others.
+* Be gracious and accept feedback and constructive criticism.
+
+These guidelines apply to all channels of communication within this GitHub repository. Please be respectful in both public channels, such as issues, and private, such as private messaging or emails.
+
+Any incidents of abuse may be reported directly to Ijwu at hmfarran@gmail.com.
```
Comment (Ijwu) [inline docs/code_of_conduct.md:11]:
```suggestion
Any incidents of abuse may be reported directly to ijwu at hmfarran@gmail.com.
```

Got sick of capital i and lowercase L looking the same....

File: docs/contributing.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+* Ensure that all changes which affect logic are covered by unit tests. 
+* Do not introduce any unit test failures/regressions.
+* Follow styling as designated in our [styling documentation](/docs/style.md).
+
+Otherwise, we tend to judge code on a case to case basis.
+
+For adding a new game to Archipelago and other documentation on how Archipelago functions, please see 
+[the docs folder](docs/) for the relevant information and feel free to ask any questions in the #archipelago-dev 
+channel in our discord.
```
Comment (Ijwu) [inline docs/contributing.md:12]:
```suggestion
channel in our [discord](https://archipelago.gg/discord).
```

**Review by Ijwu — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+title: "Bug: "
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! If this bug occurred during local generation check your 
+        Archipelago install for a log (`Archipelago/logs`) and upload it with this report, as well as all yaml files 
+        used.
```
Comment (black-sliver) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:12]:
```suggestion
        Archipelago install for a log (probably `C:\ProgramData\Archipelago\logs`) and upload it with this report, as well as all yaml files 
        used.
```

**Review by black-sliver — CHANGES_REQUESTED** [review]
I think that change may help people that don't read what installers say.

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+title: "Category: "
+labels:
+  - enhancement
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please replace `Category` in the title with what this feature will be targeting, such as Core generation,
+        website, documentation, or a game.
+        Note: this is not for requesting new games to be added. If you would like to request a game the best place to
```
Comment (SoldierofOrder) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:12]:
```suggestion
        Note: this is not for requesting new games to be added. If you would like to request a game, the best place to
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+        ask is about it is in the [discord](https://archipelago.gg/discord).
```
Comment (SoldierofOrder) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:13]:
```suggestion
        ask about it is in the [#future-game-design channel on Discord](https://archipelago.gg/discord).
```

File: .github/ISSUE_TEMPLATE/task.yaml [hunk 8]
```diff
@@ -0,0 +1,10 @@
+name: Task
+description: Submit a task to be done. If this is not targeting core it should likely be elsewhere.
```
Comment (SoldierofOrder) [inline .github/ISSUE_TEMPLATE/task.yaml:2]:
```suggestion
description: Submit a task to be done. If this is not targeting core, it should likely be elsewhere.
```

File: .github/pull_request_template.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+
+ex. "MyGame4: implement new game" or "Docs: add new guide for customizing MyGame3"
+
+## What is this fixing or adding?
+
+
+## How was this tested?
+
+
+## If this makes graphical changes please attach screenshots
```
Comment (SoldierofOrder) [inline .github/pull_request_template.md:12]:
```suggestion
## If this makes graphical changes, please attach screenshots.
````

File: README.md [hunk 10]
```diff
@@ -61,26 +61,10 @@ This project makes use of multiple other projects. We wouldn't be here without t
...
 ## Contributing
-Contributions are welcome. We have a few asks of any new contributors.
-
-* Ensure that all changes which affect logic are covered by unit tests. 
-* Do not introduce any unit test failures/regressions.
-
-Otherwise, we tend to judge code on a case to case basis. It is a generally good idea to stick to PEP-8 guidelines to ensure consistency with existing code. (And to make the linter happy.)
-
-For adding a new game to Archipelago and other documentation on how Archipelago functions, please see [the docs folder](docs/) for the relevant information and feel free to ask any questions in the #archipelago-dev channel in our discord.
+For contribution guidelines please see our [Contributing doc](/docs/contributing.md)
```
Comment (SoldierofOrder) [inline README.md:64]:
```suggestion
For contribution guidelines, please see our [Contributing doc.](/docs/contributing.md)
```

File: README.md [hunk 11]
```diff
@@ -61,26 +61,10 @@ This project makes use of multiple other projects. We wouldn't be here without t
...
 
 ## FAQ
-For frequently asked questions see the website's [FAQ Page](https://archipelago.gg/faq/en/)
+For Frequently asked questions please see the website's [FAQ Page](https://archipelago.gg/faq/en/)
```
Comment (SoldierofOrder) [inline README.md:67]:
```suggestion
For Frequently asked questions, please see the website's [FAQ Page.](https://archipelago.gg/faq/en/)
```

File: README.md [hunk 12]
```diff
@@ -61,26 +61,10 @@ This project makes use of multiple other projects. We wouldn't be here without t
...
-
-* Be welcoming and inclusive in tone and language.
-* Be respectful of others and their abilities.
-* Show empathy when speaking with others.
-* Be gracious and accept feedback and constructive criticism.
-
-These guidelines apply to all channels of communication within this GitHub repository. Please be respectful in both public channels, such as issues, and private, such as private messaging or emails.
-
-Any incidents of abuse may be reported directly to Ijwu at hmfarran@gmail.com.
+Please refer to our [code of conduct](/docs/code_of_conduct.md)
```
Comment (SoldierofOrder) [inline README.md:70]:
```suggestion
Please refer to our [code of conduct.](/docs/code_of_conduct.md)
```

File: docs/code_of_conduct.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
+# Code of Conduct
+We conduct ourselves openly and inclusively here. Please do not contribute to an environment which makes other people uncomfortable. This means that we expect all contributors or participants here to:
+
+* Be welcoming and inclusive in tone and language.
+* Be respectful of others and their abilities.
+* Show empathy when speaking with others.
+* Be gracious and accept feedback and constructive criticism.
+
+These guidelines apply to all channels of communication within this GitHub repository. Please be respectful in both public channels, such as issues, and private, such as private messaging or emails.
```
Comment (SoldierofOrder) [inline docs/code_of_conduct.md:9]:
```suggestion
These guidelines apply to all channels of communication within this GitHub repository. Please be respectful in both public channels, such as issues, and private ones, such as private messaging or emails.
```

File: docs/contributing.md [hunk 14]
```diff
@@ -0,0 +1,12 @@
+# Contributing
+Contributions are welcome. We have a few asks of any new contributors.
```
Comment (SoldierofOrder) [inline docs/contributing.md:2]:
```suggestion
Contributions are welcome. We have a few requests of any new contributors.
```
Maybe the noun "ask" is more common in circles I don't frequent, but I think "request" is more appropriate for this context.

File: docs/contributing.md [hunk 15]
```diff
@@ -0,0 +1,12 @@
...
+
+* Ensure that all changes which affect logic are covered by unit tests. 
+* Do not introduce any unit test failures/regressions.
+* Follow styling as designated in our [styling documentation](/docs/style.md).
+
+Otherwise, we tend to judge code on a case to case basis.
+
+For adding a new game to Archipelago and other documentation on how Archipelago functions, please see 
+[the docs folder](docs/) for the relevant information and feel free to ask any questions in the #archipelago-dev
```
Comment (SoldierofOrder) [inline docs/contributing.md:11]:
```suggestion
[the docs folder](docs/) for relevant information. Feel free to ask any questions in the #archipelago-dev 
```

File: docs/contributing.md [hunk 16]
```diff
@@ -0,0 +1,12 @@
...
+
+* Ensure that all changes which affect logic are covered by unit tests. 
+* Do not introduce any unit test failures/regressions.
+* Follow styling as designated in our [styling documentation](/docs/style.md).
+
+Otherwise, we tend to judge code on a case to case basis.
+
+For adding a new game to Archipelago and other documentation on how Archipelago functions, please see 
+[the docs folder](docs/) for the relevant information and feel free to ask any questions in the #archipelago-dev 
+channel in our [discord](https://archipelago.gg/discord).
```
Comment (SoldierofOrder) [inline docs/contributing.md:12]:
```suggestion
channel in our [Discord](https://archipelago.gg/discord).
```

**Review by SoldierofOrder — COMMENTED** [review]
I may not know how these files interact with GitHub, but I sure do like nitpicking punctuation and diction. :P

File: docs/code_of_conduct.md (hunk 13 above)
Comment (alwaysintreble) [reply docs/code_of_conduct.md:9]: As the code of conduct and contributing docs weren't written by me I'll leave the decision of those to Ijwu

**Review by SoldierofOrder — APPROVED** [review]
Looks great! :)

File: docs/contributing.md (hunk 15 above)
Comment (Ijwu) [reply docs/contributing.md:11]: Not merging that one. I don't like dropping articles if I can help it.

**Review by Ijwu — APPROVED** [review]

**Review by black-sliver — APPROVED** [review]
* Definitely in favour of splitting out readme parts to places where github points people to.
* The links in the readme lgtm
* Language lgtm
* How well the templates will be received can only be tested in production :-)

**Pull request merged**